lxc/main.go

```diff
@@ -189,24 +189,16 @@ func expandAlias(config *lxd.Config, origArgs []string) ([]string, bool) {
 	aliasKey := []string{}
 	aliasValue := []string{}
 
+FindAlias:
```

Comment:
Sorry to be a negative nancy here, but can we use the "matches := false" trick as before, instead of this goto-style syntax? I think basically we just want to switch it to "matches := true" by default and switch it to false if we find one that doesn't match (i.e. logical and instead of logical or).